knowledge-base/getting-telerik-reportViewer-is-not-a-function-console-error.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Getting telerik_reportViewer is not a function console error
+description: Upon running the report viewer, an error occurs in the browser console.
+type: troubleshooting
+page_title: Getting telerik_reportViewer is not a function console error
+slug: getting-telerik-reportViewer-is-not-a-function-console-error
+position: 
+tags: HTML5Viewer
+ticketid: 1410720
+res_type: kb
+---
+
+## Environment
+<table>
+    <tbody>
+	    <tr>
+	    	<td>Product</td>
+	    	<td>Progress® Telerik® Reporting</td>
+	    </tr>
+	    <tr>
+	    	<td>Project Type</td>
+	    	<td>All web-based viewers</td>
```

Comment:
Not that important, but since the viewers are not projects, maybe change the category to Report Viewer Type?